zkevm-circuits/src/evm_circuit/execution/error_invalid_jump.rs

```diff
@@ -95,6 +97,14 @@ impl<F: Field> ExecutionGadget<F> for ErrorInvalidJumpGadget<F> {
 
         cb.call_context_lookup(false.expr(), None, CallContextFieldTag::IsSuccess, 0.expr());
 
+        // constrain RwCounterEndOfReversion
```

Comment:
Move this to above line 143?

Reply:
done in [faa3721](https://github.com/privacy-scaling-explorations/zkevm-circuits/pull/1030/commits/faa372133d9079ff17be8f7346c059016dacb82a)